cmd/cloud_sql_proxy/proxy.go

```diff
@@ -238,6 +238,7 @@ func parseInstanceConfig(dir, instance string, cl *http.Client) (instanceConfig,
 		}
 	} else {
 		sql, err := sqladmin.New(cl)
+		sql.BasePath = *host
```

Comment:
Do not access fields on 'sql' until checking the 'err': 'sql' would be no, and this would panic.